### ISSUE TITLE
one hyper-parameter representation error

### ISSUE THREAD

**Issue by zhuixunforever** [NVlabs/geomapnet#31]
In [Hyperparameter](https://github.com/NVlabs/geomapnet/blob/master/scripts/configs/README.md#hyperparameters) says，beta is the initial value of Beta in Eq. 3 of the paper.
but in code, beta seem to be 'gamma' of absolute pose, and gamma seem to be 'gamma' of relative pose.
In train.py: 
> saq = section.getfloat('beta')
srq = section.getfloat('gamma')

In common/criterion.py:

> #absolute pose loss
abs_loss =\
      torch.exp(-self.sax) * self.t_loss_fn(pred.view(-1, *s[2:])[:, :3],
                                            targ.view(-1, *s[2:])[:, :3]) + \
      self.sax + \
      torch.exp(-self.saq) * self.q_loss_fn(pred.view(-1, *s[2:])[:, 3:],
                                            targ.view(-1, *s[2:])[:, 3:]) + \
      self.saq
#VO loss
vo_loss = \
      torch.exp(-self.srx) * self.t_loss_fn(pred_vos.view(-1, *s[2:])[:, :3],
                                            targ_vos.view(-1, *s[2:])[:, :3]) + \
      self.srx + \
      torch.exp(-self.srq) * self.q_loss_fn(pred_vos.view(-1, *s[2:])[:, 3:],
                                            targ_vos.view(-1, *s[2:])[:, 3:]) + \
      self.srq

**Comment by samarth-robo** [comment]
Hi @zhuixunforever you are right, thanks for pointing it out. Fixed in 6b350187e8b4be11c939bd5869c45abd9c814db4.
